lib/measures/urban_geometry_creation/tests/urban_geometry_creation_test.rb

```diff
@@ -44,34 +45,37 @@ class UrbanGeometryCreationTest < MiniTest::Unit::TestCase
   
   def test_is_shadowed
   
+    geojson_gem = URBANopt::GeoJSON::GeoJSON.new
     meas = UrbanGeometryCreation.new
     meas.origin_lat_lon = OpenStudio::PointLatLon.new(40, -120, 0)
 
     # y is north, x is east, z is up
     
     # points on ground
-    assert(!meas.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(10, 0, 0))) # West
-    assert(!meas.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(Math.sqrt(50), -Math.sqrt(50), 0)))  # South West
-    assert(!meas.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(0, -10, 0))) # South
-    assert(!meas.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(-Math.sqrt(50), -Math.sqrt(50), 0))) # South East
-    assert(!meas.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(-10, 0, 0))) # East
-    assert(!meas.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(-Math.sqrt(50), Math.sqrt(50), 0))) # North East
-    assert(!meas.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(0, 10, 0))) # North
-    assert(!meas.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(Math.sqrt(50), Math.sqrt(50), 0))) # North West
+    assert(!geojson_gem.point_is_shadowed(OpenStudio::Point3d.new(0, 0, 0), OpenStudio::Point3d.new(10, 0, 0), meas.origin_lat_lon)) # West
```

Comment:
This file is specific to this measure only, it should be used to test this measure.  General tests for GeoJSON Gem methods should go in a spec file.